templates/typescript_gapic/.jsdoc.js.njk

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright 2018 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+module.exports = {
+  opts: {
+    package: './package.json',
+    template: './node_modules/jsdoc-fresh',
+    recurse: true,
+    verbose: true,
+    destination: './docs/'
+  },
+  plugins: [
+    'plugins/markdown'
+  ],
+  source: {
+    include: [
+      'build/src'
```

Comment:
Can we look at `src` instead and include all `.ts`?  (I'm not sure it can work with TypeScript source, but can we try?)

Reply:
I am going to guess the parser will choke on TypeScript syntax, but it's worth trying.

Reply:
yes, it's unable to parse the ts file and throws error, not sure if there's other way to include `.ts` file.